### ISSUE TITLE
LANGUAGE: sv (Swedish)

### ISSUE THREAD

**Issue by HenrikBengtsson** [MichaelChirico/potools#239]
Here's the language info for <https://l10n.gnome.org/teams/sv>:

```r
> potools::translate_package(languages = "sv")
Generating en@quot translations
'sv' is not a known language. Please help supply some metadata about it. You can check https://l10n.gnome.org/teams/<language>
How would you refer to this language in English?
Swedish
How would you refer to this language in the language itself?
Svenska
How many pluralizations are there for this language [nplurals]?
2
What is the rule for deciding which plural applies as a function of n [plural]?
(n!=1)
Thanks! Please file an issue on GitHub to get this language recognized permanently
```

**Comment by MichaelChirico** [comment]
Committed straight to master by accident... PTAL: ec0ff9bda7b6a605cc8aa9e93d7e357c379bbb50

Thanks!

**Comment by HenrikBengtsson** [comment]
Thanks, but there's a typo: `Swedith`

**Comment by HenrikBengtsson** [comment]
BTW, in Swedish, "Svenska" does not have to be capitalized, i.e. in Swedish we'd use "svenska".  I'm sure it's the same in other languages too, e.g. Danish/dansk.

When I was prompted for this, that was not clear to me, so I had to peek into the code, and then I found `potools:::.potools$KNOWN_LANGUAGES` and followed the standard used there.  Is proper capitalization important for the `full_name_native` field?
